src/lib/debug.ts

```diff
@@ -0,0 +1,37 @@
+import { ChakraProps, ThemeTypings } from '@chakra-ui/react';
+
+export const debugStyles = ({
+  label = 'Debug',
+  colorScheme = 'green',
+  hideLabel = false,
+  hideBackground = true,
+}: {
+  label?: string;
+  colorScheme?: ThemeTypings['colorSchemes'];
+  hideLabel?: boolean;
+  hideBackground?: boolean;
+} = {}): ChakraProps => {
+  if (process.env.NODE_ENV !== 'development') return {};
```

Comment:
You will need to call `env` from `env.mjs` to avoid strange behavior (and for tests to turn green).

Reply:
```suggestion
  if (env.NODE_ENV !== 'development') return {};
```